projects/ngx-maplibre-gl/src/lib/popup/popup.component.spec.ts

```diff
@@ -0,0 +1,76 @@
+import { Component } from '@angular/core';
+import { ComponentFixture, TestBed, waitForAsync } from '@angular/core/testing';
+import { of } from 'rxjs';
+import { MapService } from '../map/map.service';
+import { MarkerComponent } from '../marker/marker.component';
+import { PopupComponent } from './popup.component';
+
+const getMapServiceStub = () =>
+  jasmine.createSpyObj(
+    [
+      'addMarker',
+      'removeMarker',
+      'removePopupFromMap',
+      'createPopup',
+      'addPopupToMarker'
+    ],
+    {
+      mapCreated$: of(true),
+    }
+  );
```

Comment:
just wandering, can you use ts-stubber with jasmine.createSpy?

Reply:
What works be the benefit in this case? The downside is obviously another dependency to maintain...